src/components/displayDocuments/documentCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { QrCode } from "./qrCode";
+
+const CardContainer = styled.div`
+  background: #ffffff;
+  padding: 30px;
+  width: 328px;
```

Comment:
leave it to the container to decide about width and height => this one should just take 100% of what's available